js/scripts.js

```diff
@@ -0,0 +1,135 @@
+$(function() {
+
+// -------------------------------------------------------------------------------------------------
+
+    var board = {
+        name: 'Kanban Board',
+        addColumn: function(column) {
+            this.$element.append(column.$element);
+            initSortable();
+        },
+        $element: $('#board .column-container')
+    };
+
+    $('.create-column')
+        .click(() => {
+            var name = prompt('Enter a column name');
+            var column = new Column(name);
+            board.addColumn(column);
+        });
+
+// -------------------------------------------------------------------------------------------------
+
+    function initSortable() {
+        $('.column-card-list').sortable({
+            connectWith: '.column-card-list',
+            placeholder: 'card-placeholder'
+        }).disableSelection();
+    }
+
+    function randomString() {
+        var chars = '0123456789abcdefghiklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXTZ';
+        var str = '';
+        for (i = 0; i < 10; i++) {
+            str += chars.charAt(Math.floor(Math.random() * chars.length));
+        }
+        return str;
+    }
+
+// -------------------------------------------------------------------------------------------------
+
+    function Column(name) {
```

Comment:
możesz poczytać o https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Classes i spróbować użyć klasy zamiast funkcji, ale to również bardziej jako zadanie dodatkowe :)